cli/utilities/containers.py

```diff
@@ -113,17 +121,33 @@ def run(
         if detach:
             cmd += " -d"
 
+        if interactive:
+            cmd += " -i"
+
+        if tty:
+            cmd += " -t"
+
         if rm:
-            cmd += f" --rm {rm}"
+            cmd += " --rm"
+
+        if user:
+            cmd += f" --user {user}"
+
+        if entry_point:
+            cmd += f" --entrypoint {entry_point}"
 
         if image:
             cmd += f" {image}"
 
         if cmds:
             cmd += f" {cmds}"
 
-        if self.execute(sudo=True, long_running=True, cmd=cmd):
+        out = self.execute(sudo=True, long_running=True, cmd=cmd)
+        if not out:
             raise ContainerRegistryError(f"Failed to run {cmd} on container")
+        if isinstance(out, tuple):
```

Comment:
This is incorrect, when `long_running` set to `True`, it won't return tuple

Reply:
Made the correction.